test/test-resolvers.js

```diff
@@ -118,133 +117,6 @@ Test('memoize()', (t) => {
   });
 });
 
-Test('transformResolvers()', (t) => {
-  t.test('exclusions argument is undefined', (st) => {
-    const resolvers = {
-      Query: {
-        foo() { }
-      }
-    }
-    const transformedResolvers = transformResolvers(resolvers);
-    st.equal(transformedResolvers, resolvers, 'object reference returned from transformResolvers is equal to input reference');
-    st.deepEqual(transformedResolvers, resolvers, 'object content returned from transformResolver is equal to input resolver object content');
-    st.end();
-  });
-
-  t.test('exclusions argument is an empty array', (st) => {
-    const resolvers = {
-      Query: {
-        foo() { }
-      }
-    }
-    const transformedResolvers = transformResolvers(resolvers, []);
-    st.equal(transformedResolvers, resolvers, 'object reference returned from transformResolvers is equal to input reference');
-    st.deepEqual(transformedResolvers, resolvers, 'object content returned from transformResolver is equal to input resolver object content');
-    st.end();
-  });
-
-  t.test(`exclude all types via '*'`, (st) => {
-    const resolvers = {
-      Query: {
-        foo() {}
-      },
-      Mutation: {
-        baz() {}
-      },
-      SomeType: {
-        bar() {}
-      }
-    };
-
-    const transformedResolvers = transformResolvers(resolvers, [['*']]);
-    st.deepEqual(transformedResolvers, {}, 'results in an empty resolver map being returned');
-    st.end();
-  });
-
-  t.test(`exclude a entire type by specifying 'Type' exclusion)`, (st) => {
-    const resolvers = {
-      Query: {
-        foo() {}
-      },
-      SomeType: {
-        bar() {}
-      }
-    };
-
-    const transformedResolvers = transformResolvers(resolvers, [['SomeType']]);
-    st.notOk(transformedResolvers.SomeType, 'entire specified type is excluded');
-    st.ok(transformedResolvers.Query.foo, 'other non-excluded type remains');
-    st.end();
-  });
-
-  t.test(`exclude an entire type by specifying 'Type.' exclusion`, (st) => {
-    const resolvers = {
-      Query: {
-        foo() {}
-      },
-      SomeType: {
-        bar() {}
-      }
-    };
-
-    const transformedResolvers = transformResolvers(resolvers, [['SomeType', '']]);
-    st.notOk(transformedResolvers.SomeType,'entire specified type is excluded');
-    st.ok(transformedResolvers.Query.foo, 'other non-excluded type remains');
-    st.end();
-  });
-
-  t.test(`exclude an entire type by specifying 'Type.*' exclusion`, (st) => {
-    const resolvers = {
-      Query: {
-        foo() {}
-      },
-      SomeType: {
-        bar() {}
-      }
-    };
-
-    const transformedResolvers = transformResolvers(resolvers, [['SomeType', '*']]);
```

Comment:
Do we have a replacement test for excluding by wildcard?

Reply:
With the removal of transformResolvers I switched over to using graphql-tools exclusion functionality in `mergeResolvers` which operates the same way as we had implemented: https://github.com/ExpediaGroup/graphql-component/pull/56/files#diff-62d4655acaeb66ee6637ad32b0c7c654R234

that being said .. the `importResolvers` test do test an exclusion but not with a wild card, so I'll add a test for that